src/Matterhorn/Command.hs

```diff
@@ -277,12 +277,20 @@ commandList =
   , Cmd "move-team-right" "Move the currently-selected team to the right in the team list" NoArg $ \_ ->
         moveCurrentTeamRight
   
-  --, Cmd "favorite-channel" "Mark the current channel as favorite/non-favorite" NoArg $ \_ -> do
-  --      tId <- use csCurrentTeamId
-  --      cId <- use (csCurrentChannelId tId)
-  --      s <- getSession
-  --    pref <- use crUserPreferences
-  --   applyPreferenceChange pref
+  , Cmd "favorite-channel" "Mark the current channel as favorite/non-favorite" NoArg $ \_ -> do
```

Comment:
Since this command implementation was temporary as you got things working, I recommend not committing changes like this in the future. The reasoning is that these changes are helpful during development, but they make patch review difficult since the reviewer needs to follow along to see whether these changes were replaced with other changes later in the patch sequence. I suggest using `git add -p` and skipping these hunks when committing. In general, I recommend only committing changes that are working, and keeping them as uncommitted until they have been finished. Let me know if it would be helpful to discuss that more; this is very important!

Reply:
Noted.